databuilder/models/dashboard_elasticsearch_document.py

```diff
@@ -5,22 +5,22 @@
 
 class DashboardESDocument(ElasticsearchDocument):
     """
-    Schema for the Search index document
+    Schema for the ES dashboard ES document
     """
     def __init__(self,
                  dashboard_group,  # type: str
                  dashboard_name,  # type: str
                  description,  # type: Union[str, None]
-                 last_reload_time,  # type: str
-                 user_id,  # type: str
-                 user_name,  # type: str
-                 tags  # type: list
+                 total_view,  # type: int
```

Comment:
Shall we use `total_usage` to make a same field name as the one from Table?
https://github.com/lyft/amundsendatabuilder/blob/master/databuilder/models/table_elasticsearch_document.py#L20

Reply:
updated